hera_pspec/pspecdata.py

```diff
@@ -2230,15 +2284,19 @@ def pspec(self, bls1, bls2, dsets, pols, n_dlys=None,
                         if verbose: print("  Computing and multiplying scalar...")
                         pv *= scalar
 
-                    # Wide bin adjustment of scalar, which is only needed for 
+                    # Wide bin adjustment of scalar, which is only needed for
                     # the diagonal norm matrix mode (i.e., norm = 'I')
                     if norm == 'I':
                         pv *= self.scalar_delay_adjustment(Gv=Gv, Hv=Hv)
 
                     # Generate the covariance matrix if error bars provided
                     if store_cov:
                         if verbose: print(" Building q_hat covariance...")
-                        cov_qv = self.cov_q_hat(key1, key2)
+                        if cov_method == 'empirical':
+                            cov_qv = self.get_unnormed_V(key1, key2)
+                            cov_qv = np.array([cov_qv for m in range(self.Ntimes)])
+                        elif cov_method == 'propagated':
+                            cov_qv = self.cov_q_hat(key1, key2)
```

Comment:
i don't understand why this stuff is new.. was this from an old PR that somehow found its way into here?

Reply:
I actually tweaked this because 

> It seems like what you've done in `sinc_downweight` is to apply a fourier filter, which is done via a discrete convolution whose functional form is dictated by a tophat in the fourier domain, or a sinc function in real space..

Minor comment: Its not technically a convolution unless we define it as a diagonal in delay space and derive it through a DFT. As implemented, the inverse of the sinc matrix is not quite translation invariant (not purely a convolution). 

> I'm just curious as to whether this is something we want to replace the `K` matrix in our `R` weighting, or have as a separate step in our QE chain. In other words, this matrix isn't really a covariance matrix, its a toeplitz matrix that is used for a discrete convolution, so not sure if its fair to replace the slot in QE that would normally be used for covariance matrices with this matrix... perhaps we can discuss this in person or on the next pspec telecon.

I think that the main reason to include this matrix in our power-spectrum stage is that its action is most easily incorporated into the calculation of power-spectrum covariances and error bars if its added as the K matrix. That said, we could propogate it into C either empirically or through direct calculation (though for the calculation, we'd need to develop some extra machinery outside of pspec).

We should discuss this on the next telecon.

Reply:
OK. This might be a little tangential to the PR. I ended up adding it because there are now two independent different ways of calculating power spectrum covariances but no way of specifying which one to use. I will get rid of these lines for now and create an issue because it really should be added as a different PR!